WordPressCom-Stats-iOS/InsightsTableViewController.m

```diff
@@ -85,7 +115,50 @@ - (NSInteger)numberOfSectionsInTableView:(UITableView *)tableView
 
 - (NSInteger)tableView:(UITableView *)tableView numberOfRowsInSection:(NSInteger)section
 {
-    return 2;
+    StatsSection statsSection = [self statsSectionForTableViewSection:section];
+    id data = [self statsDataForStatsSection:statsSection];
```

Comment:
second comment here: is `data` a `NSDictionary` instance?. Can we specify its data type and get rid of the `id`?